app/web/markdown/volunteer/ui-ux-designer.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "UI/UX Designer"
+description: "Apply to become a UI/UX Designer for Couchers.org"
+---
+
+## [Apply Here](/volunteer/form)
+
+## Position Description
+
+**This is a remote volunteer position**
+
+Couchers.org is seeking an experienced UI/UX Designer to help us build and improve upon next-generation, non-profit couch surfing platform. We need help refining and iterating on our existing Figma-based designs for web and mobile platforms. We’re looking for a candidate with substantial experience in ui/ux design of web and/or mobile apps with a modern, minimalist design. Our designs are already well-established, so much of the work will involve iteration and refinement; however, there are also completely new features and partially designed concepts that will benefit from your creative input and ui/ux design skills.
```

Comment:
Just wondering if the "substantial experience" is necessary. Maybe just "experience"? Or are you only looking for a senior designer type?

I'm just thinking seniors might be hard to come by volunteering, but maybe I'm wrong.

Reply:
Since I'm not a super senior designer myself, I'd prefer not to get applicants that are junior designers (at least not right now). With that said, they don't need to be senior designers, just a designer with some good experience. I said "substantial experience" because it is vague and not as scary as using "experience as a senior designer" or something.